cloudpickle/cloudpickle.py

```diff
@@ -385,7 +385,7 @@ def _save_subimports(self, code, top_level_dependencies):
         """
         # check if any known dependency is an imported package
         for x in top_level_dependencies:
-            if isinstance(x, types.ModuleType) and x.__package__:
+            if isinstance(x, types.ModuleType) and hasattr(x, '__package__') and x.__package__:
```

Comment:
Hi @mariusvniekerk, I just happened to see this PR and I am interested in this change. Would it make sense to add a simple test here if it is not too difficult?